client/src/stores/modules/cart.js

```diff
@@ -0,0 +1,23 @@
+function saveCart(cart) {
+  localStorage.cart = JSON.stringify(cart);
```

Comment:
Local store only accepts strings, so this converts the cart array of objects into a JSON string.